### DISCUSSION TITLE
Does tshock not respect the 'difficulty' flag in `server-config.txt`?

### DISCUSSION THREAD

**Discussion by markdouglasswork** [Pryaxis/TShock#1918] [Q&A]
Hi all, I'm running the latest prerelease of tshock at the time of this writingi: https://github.com/Pryaxis/TShock/releases/tag/v4.4.0-pre8

I'm wondering if I'm experiencing expected behavior or an issue. I've got the following `server-config.txt` being provided to the container:
```
autocreate=1
difficulty=3
maxplayers=8
worldname=Large Journey
worldpath=/var/terraria/worlds
world=/var/terraria/worlds/Large_Journey.wld
```
However, it seems to consistently create a classic difficulty world, with the other options being loaded correctly. I've also tried with the `difficulty=2` for a Master world. It looks like I can modify the worldmode after startup in TShock, so if not, I imagine I can just set something up to change it immediately after generation. I'd love to just be able to set the appropriate flag prior to generation and not worry about it, though. Am I missing something?

**Comment by hakusaro** [comment]
So TShock should follow whatever the vanilla config system uses, unless it's being overridden somehow. Can you test this without `TShockAPI.dll` in the `ServerPlugins` folder? So it's just TSAPI? That would help narrow it down.
